site/app/views/AutoGradingView.php

```diff
@@ -229,13 +229,19 @@ public function loadAutoChecks(\app\models\gradeable\GradedGradeable $graded_gra
         }
 
         $popup_css_file = $this->core->getOutput()->timestampResource($popup_css_file, 'css');
+        $server_css_file = $this->core->getOutput()->timestampResource('server.css', 'css');
         return $this->core->getOutput()->renderTwigTemplate("autograding/AutoChecks.twig", [
             "gradeable_id" => $gradeable->getId(),
             "checks" => $checks,
             "display_version" => $version->getVersion(),
             "index" => $testcase->getTestcase()->getIndex(),
             "who" => $who,
-            "popup_css_file" => $popup_css_file
+            "popup_css_file" => $popup_css_file,
+            "server_css_file" => $server_css_file,
+            "testcase_name" => $testcase->getTestcase()->getName(),
+            "publish_actions" => $testcase->getTestcase()->isPublishActions(),
+            "dispatcher_actions" => $testcase->getTestcase()->getDispatcherActions(),
+            "graphics_actions" => $testcase->getTestcase()->getGraphicsActions()
```

Comment:
Just pass in `"testcase" => $testcase->getTestcase()` as a single object into Twig and then call the methods from within Twig.